docs/development.md

```diff
@@ -343,9 +343,11 @@ You can optionally set the following variables:
 
 | Variable                   | Description                                                                                                               | Default                                                                               |
 |----------------------------|---------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
-| `E2E_CONF_FILE`            | The path of the [E2E configuration file](https://cluster-api.sigs.k8s.io/developer/e2e.html#defining-an-e2e-config-file).      | `${GOPATH}/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/config/azure-dev.yaml`  |
+| `E2E_CONF_FILE`            | The path of the [E2E configuration file](https://cluster-api.sigs.k8s.io/developer/e2e.html#defining-an-e2e-config-file). | `${GOPATH}/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/config/azure-dev.yaml` |
 | `SKIP_CLEANUP`             | Set to `true` if you do not want the bootstrap and workload clusters to be cleaned up after running E2E tests.            | `false`                                                                               |
 | `SKIP_CREATE_MGMT_CLUSTER` | Skip management cluster creation.                                                                                         | `false`                                                                               |
+| `LOCAL_ONLY`               | Only use Kind local registry and run the subset of tests which don't require a remotely pushed controller image.          | `false`                                                                               |
+| `REGISTRY`                 | Registry to push the controller image. If `LOCAL_ONLY=true`, this is optional.                                            | `capzci.azurecr.io/ci-e2e`                                                            |
 
```

Comment:
is there a default between LOCAL_ONLY and REGISTRY?

Reply:
I'm not sure I understand the question.

With `LOCAL_ONLY=true`, the script will override the REGISTRY with the local kind registry, `localhost:5000` and push images there. Otherwise, the REGISTRY must be able to be pushed to, and if empty will default to `capzci.azurecr.io/ci-e2e`.

I'm happy to change the flow here. Let me know how you'd like it to behave.

Reply:
I meant to ask if the user doesn't set anything, do we assume a default value of LOCAL_ONLY?

Reply:
ok, never mind, I get it know, didn't see the last column in the table, sorry :) all good